phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/managers/CurrencyManager.kt

```diff
@@ -647,7 +647,6 @@ class CurrencyManager(
         //
         val http = HttpClient {
             engine {
-                threadsCount = 1
```

Comment:
@robbiehanson This option has been deprecated at the error level and should not be used anymore, they recommend to use `dispatcher` instead. But the whole issue may have been fixed altogether in a previous version ? This needs some investigation.

Reply:
Yes, it looks like the original issue was fixed in Ktor v2.0.2. From the [release notes](https://github.com/ktorio/ktor/releases/tag/2.0.2):

> "Darwin engine: Client connection is closed after each request ([KTOR-4145](https://youtrack.jetbrains.com/issue/KTOR-4145))"

Which was a similar issue the what I reported in [KTOR-3362](https://youtrack.jetbrains.com/issue/KTOR-3362/IosClientEngine-should-use-a-shared-NSURLSession)

Also, in investigating this I discovered that api.coindesk.com appears to be completely shutdown. As in, I don't even see a DNS entry for it anymore. I'll create a new ticket for it.